recipes/hdf5/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [ `uname -m` == ppc64le ]; then
+    B="--build=ppc64le-linux"
+fi
+
+./configure $B --prefix=$PREFIX \
+    --enable-linux-lfs --with-zlib \
+    --with-pthread=yes  --enable-cxx --with-default-plugindir=$PREFIX/lib/hdf5/plugin
```

Comment:
I tried using cmake, but the resulting binaries caused the python interpreter to segfault when h5py was imported under certain circumstances.  I'm not sure they've worked all the kinks out of their cmake scripts.  Their official binaries seem to still be built using this method.
See https://github.com/ContinuumIO/anaconda-issues/issues/687